src/WPF/ArcGISRuntime.WPF.Viewer/ArcGISLocalServer.AGSDeployment

```diff
@@ -7,47 +7,47 @@ Only include these if your setup is not installing the Microsoft redistribution
   <!--The Local Server comprises a set of components which enable the use of local dynamic map and feature services based on Map Packages plus core components required for searching and querying data.
 It also provides core support for geocoding and geoprocessing.-->
   <!--Map and tile packages must be created with ArcGIS Runtime support enabled.-->
-  <Package name="Local Server" enabled="true">
+  <Package name="Local Server" enabled="false">
```

Comment:
@nCastle1 I recommend only setting the top level to `false`, and leaving all sub-packages set to `true`. It's then a much simpler change to get the LocalServer samples working.